nextcloud_install_production.sh

```diff
@@ -136,7 +141,7 @@ if [ -z "$PROVISIONING" ]
 then
     msg_box "Your current download repository is $REPO"
 fi
-if [ -n "$PROVISIONING" ] || yesno_box_yes "Do you want use http://archive.ubuntu.com as repository for this server?"
+if [ -n "$PROVISIONING" ] || yesno_box_no "Do you want use http://archive.ubuntu.com as repository for this server?"
```

Comment:
This change doesn't have really much todo with the new flag, I just think that since this is set by the provisioning variable automatically, it doesn't have to be the default during the setup anymore, since most likely you already have the best mirror available for you when also setting up ubuntu before.